tests/importer/importer.cpp

```diff
@@ -0,0 +1,766 @@
+/*
+Copyright libCellML Contributors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License->
+You may obtain a copy of the License at
+
+    http://www.apache->org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License->
+*/
+
+#include "gtest/gtest.h"
+
+#include <libcellml>
+
+#include "test_utils.h"
+
+TEST(Importer, create)
+{
+    auto importer = libcellml::Importer::create();
+    EXPECT_NE(nullptr, importer);
+}
+
+TEST(Importer, noWarningForkedImport)
+{
+    libcellml::ParserPtr p = libcellml::Parser::create();
+    libcellml::ImporterPtr importer = libcellml::Importer::create();
+    libcellml::ModelPtr model = p->parseModel(fileContents("resolveimports/forkedImport.cellml"));
+
+    EXPECT_EQ(size_t(0), p->errorCount());
+
+    EXPECT_TRUE(model->hasUnresolvedImports());
+    importer->resolveImports(model, resourcePath("resolveimports/"));
```

Comment:
It would be nice if we didn't have to provide the trailing slash, i.e. have something like:
```
importer->resolveImports(model, resourcePath("resolveimports"));
```

Reply:
This PR is about moving existing code into the `Importer`, not addressing all the niggles or changes to behaviour.  We can make issues for outstanding stuff, and I'll fix it up down the track.

Reply:
Argh, I should really try to remember that this PR is only about moving code! Shall I leave it to you to create an issue for it then once this PR has been merged in?

Reply:
I'm not convinced it is an issue ... and hopefully sooner or later we'll be able to have imports from more than one place anyway, so it's kinda moot?